src/main/java/com/salesforce/storm/spout/sideline/recipes/trigger/zookeeper/ZookeeperWatchTrigger.java

```diff
@@ -112,19 +115,11 @@
         .setDateFormat("yyyy-MM-dd HH:mm:ss")
         .create();
 
-    /**
```

Comment:
I know this seems weird, but I prefer the docs from the interface in situations like this where they're just copy pasted.